setup.cfg

```diff
@@ -278,7 +278,7 @@ disallow_incomplete_defs = True
 
 [coverage:run]
 concurrency = multiprocessing
-source = .
+source = monai
```

Comment:
Will this exclude the tests directory from coverage reporting?

Reply:
Revert the change. I try to test the change with the blossom, seems still not work, so revert it back. Thanks.